src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/SymbolType.cs

```diff
@@ -14,8 +16,20 @@ internal enum TypeKind
     }
 
     // This is a kind of Type object that will represent the compound expression of a parameter type or field type.
-    internal sealed class SymbolType : TypeInfo
+    internal sealed partial class SymbolType : TypeInfo
     {
+        #region Data Members
+        internal Type element_type = null!;
```

Comment:
The managed code under libraries should use the [repo managed naming conventions](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)

If there is a reason to use some other naming convention for Mono, it should be used under mono only.

Reply:
Maybe use fields with different names behind a readonly Property ...

Reply:
That's unnecessary complication/overhead. The north star is to have all code shared, so the runtime-specific code can be thought as temporary, and it is ok for it to be less than ideal.

Reply:
> The north star is to have all code shared, so the runtime-specific code can be thought as temporary, and it is ok for it to be less than ideal.

Good point, will rename back